src/Angor/Client/Pages/Settings.razor

```diff
@@ -707,8 +707,12 @@
 
     private async Task AddExplorer()
     {
-        if (!string.IsNullOrWhiteSpace(newExplorerLink))
+        if (string.IsNullOrWhiteSpace(newExplorerLink))
```

Comment:
what about the if bellow?

Reply:
no I mean the if statements are not aligned in one line, formatting is important for readability